.gitignore

```diff
@@ -0,0 +1,3 @@
+.dockerignore
```

Comment:
Why ignore .dockerignore? Usually you'd want to check this in so that everyone has consistent behavior.